test/test_ops.py

```diff
@@ -1155,6 +1156,68 @@ def _fn(*sample_args, **sample_kwargs):
                 graph = traced.graph_for(*inp)
                 FileCheck().check(op_name).check_not(variant_name).run(graph)
 
+class TestJitTEFuser(JitTestCase):
+    def setUp(self):
+        super(JitTestCase, self).setUp()
+        self.tensorexpr_options = TensorExprTestOptions()
+
+    def tearDown(self):
+        super(JitTestCase, self).tearDown()
+        self.tensorexpr_options.restore()
+
+    @_variant_ops(op_db)
+    def test_jit_te_fuser(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+
+        # Acquires variants to test
+        func = op.get_op()
+        method = op.get_method()
+        variants = {
+            # TODO: inplace tests currently fail, fix and add inplace variant
+            'function': func, 'method': method,
+        }
+
+        # TODO: find better way to standardize on op registration itself..
+        has_fake_function = op.name in ["resize_", 'resize_as_']
+
+        if has_fake_function:
+            variants = {'method': getattr(torch.Tensor, op.name)}
+
+        # doesn't support tracing
+        if has_fake_function:
+            return
+
+        def clone_inputs(args):
+            inputs: List[Union[torch.Tensor, List[torch.Tensor]]] = []
+
+            for arg in args:
+                if isinstance(arg, torch.Tensor):
+                    inputs.append(arg.detach().clone())
+                elif is_iterable_of_tensors(arg):
+                    inputs.append([t.detach().clone() for t in arg])
+                else:
+                    inputs.append(arg)
+
+            return inputs
+
+        for sample in samples:
+            for func_type, variant in variants.items():
+                if variant is None:
+                    continue
+
+                if is_lambda(variant):
+                    continue
+
+                trace = create_traced_fn(self, variant)
+                ref = variant(*clone_inputs((sample.input, *sample.args)), **sample.kwargs)
```

Comment:
can we also test `torch.jit.script`?

Reply:
@Krovatkin I think the idea here was that more functions support tracing than scripting, so we're starting off with only testing with tracing